preact/example-customer-account--localize--preact/extensions/localize-example/src/LocalizeExtension.jsx

```diff
@@ -0,0 +1,44 @@
+import '@shopify/ui-extensions/preact';
+import {render} from "preact";
+
+export default function() {
+  render(<LocalizeExtension />, document.body)
+}
+
+// [START localization.build-ui]
+// [START localization.format-balance]
+function LocalizeExtension() {
+  const i18n = shopify.i18n;
+  const translate = (key, replacements) => i18n.translate(key, replacements);
```

Comment:
We probably don't need this wrapper around translate?

In the other examples I have been calling `shopify` every time we use one of the APIs just to make it super clear where it comes from. So that would be calling `shopify.i18n.translate` and `shopify.i18n.formatNumber` directly each time. What do you think?